torch/profiler/_pattern_matcher.py

```diff
@@ -427,6 +434,46 @@ def match(self, event: _ProfilerEvent):
         return False
 
 
+class Conv2dBiasFollowedByBatchNorm2dPattern(Pattern):
+    '''
+    This pattern identifies if we are enabling bias in Conv2d which is followed by BatchNorm2d.
+    Bias doesn't do anything when followed by batchnorm.
+
+    Pattern:
+    nn.Module: Conv2d            | nn.Module: BatchNorm2d
+        ...
+            aten::_convolution
+                ... | aten::add_
+    # This pattern only works when using CUDA
+
+    Algorithm:
+    String match
+    '''
+
+    def __init__(self, prof: profile, should_benchmark: bool = False):
+        super().__init__(prof, should_benchmark)
+        self.name = "Enabling Bias in Conv2d Followed By BatchNorm Pattern"
+        self.description = "Detected bias enabled in Conv2d that is followed by BatchNorm2d. Please set 'bias=False' in Conv2d."
+
+    def match(self, event: _ProfilerEvent):
+        if event.name() != "aten::_convolution":
+            return False
+        if not event.children:
+            return False
+        event = event.children[-1]
+        if event.name() != "aten::add_":
+            return False
+        # This means bias=True
+        event = self.go_up_until(
+            event, lambda e: e.name().startswith("nn.Module: Conv2d"))
+        if not event:
+            return False
+        event = self.next_of(event)
```

Comment:
how would this work if forward is something like
```
x1=self.conv1(x)
x2=self.conv2(x)
x1=self.bn1(x1)
x2=self.bn2(x2)
```
Wouldn't next_of be just the next conv in this case?

Reply:
Ah. Right now I am going to narrow the scope of detection to Situation in nn.Sequential, or simpler scenarios, as we do not have tensor identities information (Only size and dtype) as of right now. But great point! I will add a todo

Reply:
But if we do have that information, we can do something like:

event = next_used_event(event.output_tensor)

Reply:
tensor identities are needed for more robust pattern detection